packages/react/src/components/organisms/HeaderSlim/index.js

```diff
@@ -4,36 +4,39 @@
  * @requires module:@massds/mayflower-assets/scss/03-organisms/header-slim
  */
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const HeaderSlim = (props) => (
+const HeaderSlim = ({
+  SkipNav,
+  SiteLogo,
+  MainNav,
+  UtilityNav
+}) => (
   <div className="ma__header_slim">
-    <div className="ma__header_slim__banner" />
+    {SkipNav && <SkipNav />}
+    <div className="ma__header_slim__utility">
+      <div className="ma__header_slim__utility-container ma__container">
+        {UtilityNav && <UtilityNav />}
+      </div>
+    </div>
     <header className="ma__header_slim__header" id="header">
-      <div className="ma__header_slim__container ma__container">
-        <div className="ma__header_slim__logo">{props.siteLogo}</div>
-        {props.children}
+      <div className="ma__header_slim__header-container ma__container">
+        <div className="ma__header_slim__logo"><SiteLogo /></div>
+        { MainNav && <div className="ma__header_slim__nav"><MainNav /></div> }
       </div>
     </header>
   </div>
 );
 
 HeaderSlim.propTypes = {
-  /** The domain you want to send users to from the site logo icon */
-  // eslint-disable-next-line consistent-return
-  siteLogo: (props, propName, componentName) => {
-    const component = props[propName];
-    const isValid = (comp) => {
-      if (typeof comp.type === 'string') {
-        return comp.type === 'SiteLogo';
-      }
-      return comp.type.name && comp.type.name === 'SiteLogo';
-    };
-    if (!component || (component && !isValid(component))) {
-      return new Error(`Invalid prop ${propName} supplied to ${componentName}. Got: ${
-        component.type.name
-      }. Validation failed.`);
-    }
-  }
+  /** A render function that renders SiteLogo component. */
+  SiteLogo: PropTypes.func.isRequired,
```

Comment:
Nit: I don't feel super strongly either way, but could see these being prefixed with `render` (e.g. `renderSiteLogo`) to make it clearer these are render function props, rather than nodes. When I first saw this, I assumed I could use the component like this (which maybe you can?):

```js
import MyLogo from "./components/MyLogo";

...

<HeaderSlim SiteLogo={MyLogo} />
```

Reply:
updated proptypes --> https://github.com/massgov/mayflower/pull/1207#issuecomment-714519911